ref/README.md

```diff
@@ -21,8 +21,32 @@ cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
 make VERBOSE=1
 ```
 
+### Building the kernel (Systems without modules)
+
+```
+rm -rf build ; mkdir build ; cd build
+export CC=<gcc | icc>
+export FC=<gfortran | ifort> 
+cmake -DCMAKE_BUILD_TYPE=<debug | release> ..
+make VERBOSE=1
+```
+
+### Additional description for build configurations 
+
+Some alterations could potentially need to be made in regards to the CC and FC compilers  
+
+Confirm and identifiy which compiler will be used on the system.  
+
+* export CC= < appropriate compiler & version  > 
+* export FC= < appropriate compiler & version  >
+
+For Example : ```export CC=gcc-10```
```

Comment:
CC needs to be the name of the compiler.  Say for example, we want to compare GNU 7 and GNU 10 with the kernel.  The compiler would be gcc in both cases, the path would be modified to choose gcc version 7 or gcc version 10.

Reply:
@LyndStringer - Not on a Mac, depending on how you installed gcc there.  On Macs, Homebrew installs gcc in `/usr/local/bin` with the version number appended to the executable.  And `gcc` actually points to the Clang compiler.  So, for Mac, you either need to play symlink tricks when you install GCC, or you need to set `CC` to the compiler that Hombrew installed, `gcc-10`.

Reply:
The documentation could spell that out a little more clearly.